src/App.js

```diff
@@ -1,21 +1,103 @@
-import React, { useState, useEffect } from 'react';
-import PokemonList from './PokemonList';
-import axios from "axios";
-import './App.css';
+import React, { useState, useEffect } from "react";
+import PokemonList from "./PokemonList";
```

Comment:
please create a folder called `components`  and put PokemonList  inside, same with Pagination...
Good code structure is 
src/
   pages/
   components/
   assets/

Reply:
Would this include the public file and node_module being in the same directory?

Reply:
@michaeltomasik I created the file structure as shown, I'm noticing that the app could not access the modules outside the src file.

Reply:
right.. he formatting of my previous message is wrong
```
src/pages/
src/components/
src/assets/
```

Reply:
restructured file directory